src/xtgeo/grid3d/grid.py

```diff
@@ -156,25 +148,16 @@ def __init__(
         self,
         gfile: Optional[Union[str, Path]] = None,
         fformat: Optional[str] = "guess",
-        initprops: Optional[List[str]] = None,
-        restartprops: Optional[List[str]] = None,
-        restartdates: Optional[List[Union[int, str]]] = None,
-        ijkrange: Optional[IJKRange] = None,
-        zerobased: Optional[bool] = False,
+        **kwargs,
```

Comment:
Generally I would refrain from using `**kwargs` in an init (not the same case as `from_file` which redirects to different functions - there it's fine), but since https://github.com/equinor/xtgeo/pull/655 will explicitly state the arguments I don't see an issue with it. Guess it wouldn't be a problem leaving it as is either? But I'll leave it up to you :)

Reply:
Yea, I understand what you mean, but the `**kwargs` relates to the init via `xtgeo.grid_from_file` which will be removed.